cluster/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
-  name: rook-ceph-cluster-mgmt
+  name: {{ if .Values.multipleOperators }}{{.Release.Name }}-{{ end }}rook-ceph-cluster-mgmt
```

Comment:
Having the `.Release.Name` in the names and as a label too seems to be a common practice for "official" Helm charts.
(Off-topic: I'm just too lazy to always think of a release name when I deploy a chart once and not twice, still it is a best practice)

Having it behind a toggle seems good, though what about the CRDs?